EIPS/eip-158.md

```diff
@@ -4,7 +4,7 @@ title: State clearing
 author: Vitalik Buterin
 type: Standards Track
 category: Core
-status: Replaced
+status: Superseded
 created: 2016-10-16
 superseded-by: 161
```

Comment:
It has a `superseded-by` field so the intent clearly was "Superseded" especially as "Replaced" is an invalid status.